### PR TITLE
Update Vagrant image and deps.

### PR DESCRIPTION
**Pull request by kadams54**
Brings Vagrant onto a current (trusty) version of Ubuntu, as well as replacing the Chris Lea PPA.

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

1. `vagrant destroy`
2. `vagrant up`
3. `vagrant ssh`
4. Change to your juju-gui directory within vagrant (e.g., `cd /vagrant/Code/juju-gui`)
5. `make clean-all devel`
6. Dink around in juju-gui long enough to verify that everything is back up and running OK.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2554/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2555/
Test PASSed.

File: Vagrantfile [hunk 1]
```diff
@@ -8,11 +8,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Every Vagrant virtual environment requires a box image to base.  Use a 64bit
   # box from the latest Ubuntu release; if it does not exist, vagrant up will
   # fetch a box and name it properly.
-  config.vm.box = "saucy64"
-
-  # The url from where the 'config.vm.box' box will be fetched if it
-  # doesn't already exist on the user's system.
-  config.vm.box_url = "http://cloud-images.ubuntu.com/vagrant/saucy/current/saucy-server-cloudimg-amd64-vagrant-disk1.box"
+  config.vm.box = "ubuntu/trusty64"
```
Comment (hatched) [inline Vagrantfile]: You said vivid in your description but this is trusty. Was the description incorrect or the code? :)

**Comment by hatched** [comment]
:+1: QA OK! Thanks for this update

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**